tests/param_api_test.py

```diff
@@ -13,11 +11,19 @@ class MyPositionsTests(unittest.TestCase):
         ('empty', '', 404, 'Not Found', '')
     ]
 
+    data_application = [
+        ('application does not exist', 9999, 400, 'Bad Request', ''),
+        ('letters', 'abc', 500, 'Internal Server Error', 'ER_BAD_FIELD_ERROR'),
+        ('space', ' ', 400, 'Bad Request', ''),
+        ('empty', '', 200, 'OK', '')
```

Comment:
So would this last set of data be considered a positive?
Is there anything additional we can test here?